src/core/annotation.js

```diff
@@ -3190,6 +3190,9 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
       this._streams.push(this.uncheckedAppearance);
     }
     this._fallbackFontDict = this.fallbackFontDict;
+    if (this.data.defaultFieldValue === null) {
+      this.data.defaultFieldValue = "Off";
+    }
```

Comment:
Would it make sense to do the same thing in the `_processRadioButton` method below, or would that perhaps be too risky without a test-case?